pkg/util/containers/filter.go

```diff
@@ -23,11 +23,18 @@ const (
 	pauseContainerOpenshift  = "image:openshift/origin-pod"
 	pauseContainerOpenshift3 = "image:.*rhel7/pod-infrastructure"
 
+	// - asia.gcr.io/google-containers/pause-amd64
+	// - gcr.io/google-containers/pause
+	// - *.gcr.io/google_containers/pause
+	// - *.jfrog.io/google_containers/pause
+	pauseContainerGoogle = "image:google(_|-)containers/pause.*"
```

Comment:
is the domain needed? i thought it was but could be mistaken

Reply:
it's to support private registries.

Reply:
I was thinking `google_container/pause` was already enough specific

Reply:
i suppose a test case in filter_test.go would be good to have either way

Reply:
@celenechang unittests added

Reply:
looks good to me!